packages/testsuite/cypress/support/containers-utils.ts

```diff
@@ -22,9 +42,49 @@ declare global {
     interface Chainable {
       /**
        * Start a Wildfly container. This typical command which needs to executed in before method in every TC.
+       * For this case use used unsecured management interface and web console doesn't require the authentication.
+       *
+       * Is used also network mode so the container is running in a network for this test and is exposed a random port
+       * for specific services provided by wildfly.
+       *
        * @category Containers util
        */
       startWildflyContainer(): Chainable<unknown>;
+      /**
+       * Start a Wildfly container wich is secured. This is not for a common use.
+       *
+       * This container is using standart "standalone.xml" and have a secured management interface. The container is
+       * running in network mode "host". To enable communication between browser (not in container),
+       * application server (in container), and Keycloak (in container). Those need to use the same network
+       * and consistent host names. This mean the wildlfly looks like it is stared directly on a host machine.
+       * There can be potention conflict for used ports so there is range from 8080 to 8180 to check the port
```

Comment:
> There can be potention conflict for used ports so there is range from 8080 to 8180 to check the port free and then is calculated port offset for start parameter (0-100).

To much details. What about the following?

-> A random port offset is used to avoid conflicts.

Reply:
updated